### DISCUSSION TITLE
Mid-year check-in - Codespaces check-in

### DISCUSSION THREAD

**Discussion by otter-computer** [community/community#59773] [Codespaces]
Hey all, welcome to the latest Codespaces check-in! I’m currently baking away in sunny Scotland’s annual heatwave (they just keep getting hotter!) ☀️ but I’m still here to bring you the latest from the community and from the Codespaces team, so here we go~

## From the Community 💬 
- Huge thanks to @SoloLeveling7seven for returning to the community with [an answer to their own issue on Codespaces getting stuck during setup](https://github.com/orgs/community/discussions/58374). There were a lot of people facing similar problems so thanks for coming back to share the solution that worked for you! 🎉
- Thanks to @spenserblack for sharing your thoughts on @max-zinn’s question on [Codespaces storage usage](https://github.com/orgs/community/discussions/59588)

### [Changes to Codespaces Access, Ownership, and Billing Settings](https://github.blog/changelog/2023-06-09-upcoming-changes-to-codespaces-access-ownership-and-billing-settings/)
GitHub Codespaces is rolling out improved access controls for organizations 🎮. These changes will provide organizations additional control over which of their organization’s members or outside collaborators are allowed to use GitHub Codespaces on private and internal repositories. For more information on what this means for your organization check out the [changelog post](https://github.blog/changelog/2023-06-09-upcoming-changes-to-codespaces-access-ownership-and-billing-settings/) on the GitHub blog.
### [Maximum Codespaces per User Policy](https://github.blog/changelog/2023-06-15-maximum-codespaces-per-user-policy/)
Organization administrators can now specify the maximum number of organization-billed codespaces that any member of the organization, or collaborator, can create. For more information check out the [changelog post](https://github.blog/changelog/2023-06-15-maximum-codespaces-per-user-policy/).

That’s all of this time! Have any thoughts on anything announced above, or spotted anyone in the community you’d love to give massive kudos to? Shout it out in the comments below 📣. Until next time~

**Comment by the-officialjosh** [comment]
Awesome changes 🚀

**Comment by rhino701** [comment]
👍 👍 👍

**Comment by Vicwxy9** [comment]
👍 👍 👍👍 👍 👍👍 👍 👍👍 👍 👍

**Comment by Binary-Ninja-007** [comment]
Good.

**Comment by Cclement00** [comment]
Bonjour je veux faire un code cmmme ce lien : https://believemy.com/login

Pouvez vous aider pour ce code.
